packages/lodestar-beacon-state-transition/src/lightclient/block.ts

```diff
@@ -0,0 +1,13 @@
+import {IBeaconConfig} from "@chainsafe/lodestar-config";
+import {Lightclient} from "@chainsafe/lodestar-types";
+import * as phase0Block from "../block";
```

Comment:
might be nice to segregate phase0-specific code to a `phase0/` directory

Reply:
> might be nice to segregate phase0-specific code to a `phase0/` directory

I would love that. But it's enormus diff and change because we didn't use exports in root index 😭

Reply:
the exports from `src/fast/` ?

Reply:
We use imports from fast and normal and they are all over the place